tracex_project/tracex/tests/test_utils.py

```diff
@@ -0,0 +1,162 @@
+"""Test cases for utils."""
+from django.db.models import Q
+from django.test import TestCase
+import pandas as pd
+
+from extraction.models import Trace
+from tracex.logic.utils import Conversion, DataFrameUtilities
+
+
+class ConversionTests(TestCase):
+    """Test cases for the conversion class inside the utils module."""
+
+    def test_prepare_df_for_xes_conversion(self):
+        """Tests if the dataframe contains the correct column name for xes conversion"""
```

Comment:
"""Tests if the dataframe contains the correct column name for xes conversion"""
```suggestion
        """Tests if the dataframe contains the correct column name for the xes conversion"""
```